### ISSUE TITLE
ARM assembler core not work in Thumb mode (gcc)

### ISSUE THREAD

**Issue by DarkMike-ru** [compuphase/pawn#47]
Hello!

I try to use amxexec_thumb2_gas.s with AMX_TOKENTHREADING option

In macro NEXT your switch to next instruction processing by load pc
`ldr pc, [r14, r11, LSL #2]`

Value extracted from amx_opcodelist table:
```
    .word   .OP_NOP
    .word   .OP_LOAD_PRI
    .word   .OP_LOAD_ALT
```
But values in this table is even, so load this value in PC switch processor in ARM mode instead of Thumb
Solution:
```
    .word   .OP_NOP + 1
    .word   .OP_LOAD_PRI + 1
    .word   .OP_LOAD_ALT + 1
```

**Comment by compuphase** [comment]
You are correct. The core was tested on a Cortex M3 microcontroller. These support Thumb **only**, and therefore simply ignore the low bit. For ARM microcontrollers that support both ARM and Thumb modes, the relevant assembler cores must be fixed.

**Comment by compuphase** [comment]
Fixed in commit 8ee4beea053a61f6e6c336a89e96ca1c1d02d7a7.
